### PR TITLE
Update human labels for ferry routes

### PR DESCRIPTION
**Pull request by amaisano**
#### Summary of changes
**Asana Ticket:** [Update display text for ferry fare cards](https://app.asana.com/0/399597520748778/1175281228582685)

Commuter Ferry + Inner Harbor Ferry updated

image

---

Before getting review, please check the following:

* [x] Does frontend functionality render and work correctly in IE?
* [x] Have we load-tested any new pages or internal API endpoints that will receive significant traffic?
* [x] Are interactive elements accessible to screen readers?
* [x] Have you checked for tech debt you can address in the area you're working in?
* [x] If this change involves routes, does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?
* [x] Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?

### CODE REVIEW

File: apps/fares/lib/format.ex [hunk 1]
```diff
@@ -83,10 +83,10 @@ defmodule Fares.Format do
   def name(:local_bus), do: "Local Bus"
   def name(:inner_express_bus), do: "Inner Express Bus"
   def name(:outer_express_bus), do: "Outer Express Bus"
-  def name(:ferry_inner_harbor), do: "Inner Harbor Ferry"
+  def name(:ferry_inner_harbor), do: "Charlestown Ferry"
```
Comment (alicia-mbta) [inline apps/fares/lib/format.ex:86]: Would it make sense to rename the atoms as well not only the strings?

Comment (amaisano) [reply apps/fares/lib/format.ex:86]: I think the atoms should resemble the GTFS system route names more than human label. That being said, not sure what those are off the top of my head but I assumed the current atoms lined up with GTFS.

Comment (alicia-mbta) [reply apps/fares/lib/format.ex:86]: Ah ok disregard then

**Comment by aayesha** [comment]
Sorry for the late review. Looks good!

**Pull request merged**